modules/mono/editor/GodotTools/GodotTools/Build/BuildManager.cs

```diff
@@ -13,9 +13,11 @@ public static class BuildManager
     {
         private static BuildInfo? _buildInProgress;
 
+        private static bool _first_build = true;
+
         public const string MsBuildIssuesFileName = "msbuild_issues.csv";
         private const string MsBuildLogFileName = "msbuild_log.txt";
-
+        
```

Comment:
This line needs to be empty with no whitespace

Reply:
also _first_build is now _firstBuild complying with the set c# standard